liferay-echo/liferay-portal
# LPD-39372 The "Make a copy" action should be "Copy to" action

> [!Note]
> There are integration tests for this action, but I added a playwright one to test the new modal due to the copy to action

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-39372)

### Changes

Chage the copy action for the folders in display pages - due to requirements

## Test scenario 

1. Create two folders
2. Go to the kebab menu of one of them
3. Click on copy to
4. Select the other folder to copy into it
5. Check that the copy has been made
6. Try it also using the page management action

[Screencast from 2024-10-17 00-19-50.webm](https://github.com/user-attachments/assets/5dd8cd85-01da-4f7f-9f21-07c6f3ff33f1)
